selftests/deployment/roles/common/tasks/aexpect.yml

```diff
@@ -0,0 +1,9 @@
+- name: Install python3-aexpect first
+  package:
+    name: "{{ item }}"
+    state: latest
```

Comment:
Maybe `state: present` ? If not, a second run of this playbook will fail because the avocado module repos have been disabled.

Reply:
You are right, good catch.